packages/editor/src/components/post-featured-image/index.js

```diff
@@ -160,83 +170,111 @@ function PostFeaturedImage( {
 						unstableFeaturedImageFlow
 						allowedTypes={ ALLOWED_MEDIA_TYPES }
 						modalClass="editor-post-featured-image__media-modal"
-						render={ ( { open } ) => (
-							<div className="editor-post-featured-image__container">
-								<Button
-									className={
-										! featuredImageId
-											? 'editor-post-featured-image__toggle'
-											: 'editor-post-featured-image__preview'
-									}
-									onClick={ open }
-									aria-label={
-										! featuredImageId
-											? null
-											: __( 'Edit or update the image' )
-									}
-									aria-describedby={
-										! featuredImageId
-											? null
-											: `editor-post-featured-image-${ featuredImageId }-describedby`
-									}
-								>
-									{ !! featuredImageId && media && (
-										<ResponsiveWrapper
-											naturalWidth={ mediaWidth }
-											naturalHeight={ mediaHeight }
-											isInline
-										>
-											<img
-												src={ mediaSourceUrl }
-												alt=""
-											/>
-										</ResponsiveWrapper>
-									) }
-									{ isLoading && <Spinner /> }
-									{ ! featuredImageId &&
-										! isLoading &&
-										( postType?.labels
-											?.set_featured_image ||
-											DEFAULT_SET_FEATURE_IMAGE_LABEL ) }
-								</Button>
-								<DropZone onFilesDrop={ onDropFiles } />
-							</div>
-						) }
-						value={ featuredImageId }
-					/>
-				</MediaUploadCheck>
-				{ !! featuredImageId && (
-					<MediaUploadCheck>
-						{ media && (
-							<MediaUpload
-								title={
-									postType?.labels?.featured_image ||
-									DEFAULT_FEATURE_IMAGE_LABEL
+						render={ ( { open: openMediaLibrary } ) => (
+							<FormFileUpload
+								accept="image/*"
+								onChange={ ( event ) =>
+									onDropFiles( event.target.files )
 								}
-								onSelect={ onUpdateImage }
-								unstableFeaturedImageFlow
-								allowedTypes={ ALLOWED_MEDIA_TYPES }
-								modalClass="editor-post-featured-image__media-modal"
-								render={ ( { open } ) => (
-									<Button
-										onClick={ open }
-										variant="secondary"
-									>
-										{ __( 'Replace Image' ) }
-									</Button>
+								render={ ( { openFileDialog } ) => (
+									<div className="editor-post-featured-image__container">
+										<Dropdown
+											className="editor-post-featured-image__dropdown"
+											popoverProps={ {
+												placement: 'left-start',
```

Comment:
This doesn't look the greatest on mobile. Maybe should consider `'overlay`' or `'bottom'` when the viewport is small.

Reply:
Here's what I'm seeing:

<img width="300" alt="Screenshot 2023-05-03 at 4 24 31 pm" src="https://user-images.githubusercontent.com/6458278/235845017-45aed78a-f60b-476b-8b68-fec13d4a141f.png">

Would it be acceptable to hide the popover in narrower viewports? Or just leave it as it is in trunk?

<img width="300" alt="Screenshot 2023-05-03 at 4 25 30 pm" src="https://user-images.githubusercontent.com/6458278/235845265-236bfe53-809a-41ea-a6b4-3d8f13a7c548.png">

Reply:
> Would it be acceptable to hide the popover in narrower viewports and have a modified version of what's on trunk?

Actually, now that I think of it, the replace image button is redundant since the image is a button and triggers the same action.

The remove link could just be a ❌  icon too.

🤷 What do I know.

Reply:
For narrower viewports maybe we could do something like this (or with the two buttons below the image, but looking the same)

<img width="928" alt="image" src="https://user-images.githubusercontent.com/548849/236022010-441a45e7-acd7-4a3f-b066-4161728df0a9.png">